src/assets/theme/overrides/MuiAutocomplete.tsx

```diff
@@ -0,0 +1,6 @@
+export default {
+  root: {
+    backgroundColor: 'white',
+    'border-radius': '4px',
+  },
+};
```

Comment:
I'm not sure if this was intended, but the autocomplete box and the email input now have a black border on hover

![Kapture 2020-07-20 at 12 00 32](https://user-images.githubusercontent.com/114084/87975621-b6d10e00-ca80-11ea-8424-47c419aa97f1.gif)

Reply:
yeah, this black hover is the default state on the material ui autocomplete component and i spent way tooooo much time trying to figure out how to get rid of it. eventually threw in the towel and just made the email input match